Release 2.10.0. Final (Ready to Merge)

Furtif opened the pull request:
## Ready to Merge

Furtif committed:
Release 2.10.0 Preview

Furtif committed:
Release 2.10.0.

Furtif committed:
Release 2.10.0.

CDAGaming commented on POGOProtos.csproj:
```diff
@@ -296,7 +298,7 @@
     <Compile Include="Networking\Responses\UseItemXpBoostResponse.g.cs" />
     <Compile Include="Networking\Responses\VerifyChallengeResponse.g.cs" />
-    <Compile Include="Properties\AssemblyInfo.cs" />
+	<Compile Include="Properties\AssemblyInfo.cs" />
```
Please Fix this Indent

Furtif replied:
that is to say ? Old protos https://github.com/Necrobot-Private/POGOProtos/blob/master/Properties/AssemblyInfo.cs 2.9.1 real 2.9.2 (ident a nuget pack) this new is 2.10.0!!!

Furtif committed:
.

Furtif commented:
It's aligned that's what you're talking about ???
Anyway I just test the version is not valid by AeonLucid yet ...
Maybe need to add files or delete a to see when it will validate all this

Furtif committed:
Release 2.10.0
https://github.com/AeonLucid/POGOProtos/commit/64ec9e0b2d143bcd6125fd05ab549520afc0ee44
https://github.com/AeonLucid/POGOProtos/commit/cb71305acc9823e545177b64d7b3d3c26763d239
